common/component_checker.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2019 Intel Corporation
```

Comment:
I see no pylint run for this script here: http://mediasdk.intel.com/buildbot/#/builders/17/builds/1097

Reply:
File was not changed in last commit, see pre-commit check for first commit http://mediasdk.intel.com/buildbot/#/builders/17/builds/1096